gateway/build.gradle.kts

```
@@ -0,0 +1,25 @@
+plugins {
+    `spring-conventions`
+    kotlin("kapt")
+}
+
+dependencies {
+    implementation("org.mapstruct:mapstruct:1.6.0")
+    implementation(project(":internal-api"))
+    implementation("org.springframework.boot:spring-boot-starter-web:3.3.2")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("io.nats:jnats:2.16.14")
+    implementation("io.projectreactor:reactor-core:3.6.10")
+    implementation(project(":core")) {
+        exclude(group = "org.springframework.boot", module = "spring-boot-starter-security")
+    }
```

```suggestion
    implementation(project(":internal-api"))
        implementation(project(":core")) {
        exclude(group = "org.springframework.boot", module = "spring-boot-starter-security")
    }
    implementation("org.mapstruct:mapstruct:1.6.0")
    implementation("org.springframework.boot:spring-boot-starter-web:3.3.2")
    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
    implementation("io.nats:jnats:2.16.14")
    implementation("io.projectreactor:reactor-core:3.6.10")
```

keep your dependencies structured